Plugins/NativeCamera/Source/Apple/NativeCameraImpl.mm

```diff
@@ -463,7 +468,9 @@ fragment float4 fragmentShader(RasterizerData in [[stage_in]],
 
     void Camera::Impl::Close()
     {
-        m_implData.reset(new ImplData);
+        [m_implData->cameraTextureDelegate reset];
+        m_implData.reset();
+        m_implData = std::make_unique<ImplData>();
```

Comment:
We should probably change `m_implData` back to `std::unique_ptr` in the header now that it's not shared anymore.

Reply:
Good call!